conf/config.yaml.sample

```diff
@@ -64,11 +64,41 @@ cortx:
     transport_type: libfab
     ios:
       endpoints:
-      - motr://data1-node1.cortx-cluster.com:3232
+      - tcp://data-node1:28002
+      - tcp://data-node1:28003
+      - tcp://data-node2:28002
+      - tcp://data-node2:28003
+      - tcp://data-node3:28002
+      - tcp://data-node3:28003
+    confd:
+      endpoints:
+      - tcp://data-node1:28004
+      - tcp://data-node2:28004
+      - tcp://data-node3:28004
+    s3_instances: 1 # cortx s3 servers
```

Comment:
s3_instances: 1 # cortx s3 servers
    s3:
      endpoints:
      - tcp://server-node1:28002
      - tcp://server-node2:28002
      - tcp://server-node3:28002
      
      
      As s3server instances are only one here change it to above and similarly at other places.

Reply:
Updated s3 endpoints.